src/Core/Probers/MultiByte/UTF8Prober.cs

```diff
@@ -107,11 +119,17 @@ public override float GetConfidence(StringBuilder status = null)
         {
             float unlike = 0.99f;
             float confidence;
+            var mbCharRatio = 0;
+            var nonBasciAsciiLen = fullLen - basicAsciiLen;
+            if (nonBasciAsciiLen > 0)
+            {
+                mbCharRatio = mbCharLen / nonBasciAsciiLen;
+            }
 
-            if (numOfMBChar < 6)
+            if (numOfMBChar < 6 && mbCharRatio <= 0.6)

```

Comment:
Hello! Did I understand correctly that `mbCharRatio` is the result of integer division? Why is the comparison with a real number?

Reply:
I believe it is a mistake since the code is rewritten from jschardet . I will fix it.

Reply:
Now it's fixed